torch/csrc/autograd/python_variable.cpp

```diff
@@ -525,9 +527,9 @@ static PyObject* THPVariable_as_subclass(PyObject* _self, PyObject* args, PyObje
 static PyObject* THPVariable_make_subclass(PyObject* _ignored, PyObject* args, PyObject* kwargs) {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
-    "_make_subclass(PyObject* cls, Tensor data, bool require_grad=False)",
+    "_make_subclass(PyObject* cls, Tensor data, bool require_grad=False, bool use_python_dispatch=False)",
```

Comment:
This is a misnomer; what you're really doing is making it possible to intercept strides

Reply:
Thanks for the comment - to stay in line with the C++ naming, I've changed this to `use_custom_strides`. Happy to change to any names others might prefer!